ocs_ci/utility/framework/base_initializer.py

```
@@ -0,0 +1,205 @@
+
+        Args:
+            kubeconfig_path (str): Path to kubeconfig file
+
+        Raises:
+            ClusterNotAccessibleError: If the cluster is inaccessible
+        """
+        logger.info("Testing access to cluster with %s", kubeconfig_path)
+        if not os.path.isfile(kubeconfig_path):
+            raise ClusterNotAccessibleError(
+                "The kubeconfig file %s doesn't exist!", kubeconfig_path
+            )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)